app/src/main/java/com/sun/unsplash_01/data/repository/HomeRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.unsplash_01.data.repository
+
+import com.sun.unsplash_01.data.source.DataSource
+import com.sun.unsplash_01.utils.Constant
+
+class HomeRepository(private val remote: DataSource.Remote) {
```

Comment:
Repository dùng chùng cho toàn app chứ không phải chỉ dùng màn hình home.
Đặt lại tên e